specs/mediawiki/v1/content.yaml

```diff
@@ -75,7 +70,7 @@ paths:
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}/latest
 
-  /{module:page}/{title}/html/:
+  /{module:page}/html/{title}/:
```

Comment:
Might each be useful in different cases?

If so, we could support both (and be ready to support related cases in the future) with the [`Accept` header](http://tools.ietf.org/html/rfc2616#page-100).